web/src/beta/features/Dashboard/ContentsContainer/Projects/index.tsx

```diff
@@ -0,0 +1,197 @@
+import { FC, useRef, MouseEvent } from "react";
+
+import { Loading, PopupMenuItem, Typography } from "@reearth/beta/lib/reearth-ui";
+import { onScrollToBottom } from "@reearth/beta/utils/infinite-scroll";
+import { useT } from "@reearth/services/i18n";
+import { styled, useTheme } from "@reearth/services/theme";
+
+import { TabProps } from "..";
+import { Project as ProjectType } from "../../type";
+import { CommonHeader } from "../header";
+
+import useHooks from "./hooks";
+import { Project } from "./project";
+import { ProjectModal } from "./projectModal";
+
+const options = [{ value: "date", label: "Latest modified" }];
+
+export type LayoutProps = {
+  project: ProjectType;
+  popupMenu: PopupMenuItem[];
+  selectedProjectId?: string;
+  isStarred?: boolean;
+  isEditing?: boolean;
+  projectName?: string;
+  isHovered?: boolean;
+  onStarClick?: (e: MouseEvent<Element>, projectId: string) => void;
+  onProjectOpen?: () => void;
+  onProjectUpdate?: (project: ProjectType, projectId: string) => void;
+  onProjectSelect?: () => void;
+  onBlur?: () => void;
+  onChange?: (value: string) => void;
+  onDoubleClick?: () => void;
+  onHover?: (v: boolean) => void;
```

Comment:
Could you update the namings of these events? Consider it's prop of `Layout` current naming will be understood as onLayoutBlur onLayoutChange etc. which seems confused.

Reply:
And seems this type doesn't been used in this component, maybe better define it in an independent type file?